website/docs/config-reference/codegen-config.md

```diff
@@ -105,17 +125,24 @@ generates:
       - typescript-operations
 ```
 
-You can load an `.env` file by adding the `-r dotenv/config` option to your CLI command.
+You can load a `.env` file by adding the `-r dotenv/config` option to your CLI command.
```

Comment:
Did it more correct ?

Reply:
Not sure about the grammatical rule here 😅